web/client/src/components/validation/TokenValidation.jsx

```diff
@@ -21,7 +21,6 @@ import "../../styles/form.css";
 import "../../styles/validation.css";
 import { validateToken } from "../../util/apiWrapper";
 import { makeStyles } from "@material-ui/core/styles";
-
```

Comment:
Add this new line back between imports and arrow function

Reply:
Done!